client/src/main/java/com/alibaba/nacos/client/naming/core/HostReactor.java

```diff
@@ -411,7 +412,7 @@ public void run() {
                     return;
                 }
                 
-                delayTime = serviceObj.getCacheMillis();
+                delayTime = serviceObj.getCacheMillis() + Math.min(serviceObj.getFailCount() * 1000L, 10000L);
```

Comment:
The exception delay time should be calculate in `catch`
And the algorithm should be enhance. Such as double time then last time. And the max delay time is not more than 60s.